app/views/shop/products/_form.html.haml

```diff
@@ -1,8 +1,8 @@
 %form{action: main_app.cart_path}
   %products{"ng-controller" => "ProductsCtrl", "ng-init" => "refreshStaleData()", "ng-show" => "order_cycle.order_cycle_id != null", "ng-cloak" => true }
 
-    = render partial: "shop/products/searchbar"
-
+      = render partial: "shop/products/searchbar"
+      
```

Comment:
Hello Robin, I wonder if you have a problem with your local master...
to clean up your local master you can do this:

git checkout master
git reset --hard upstream/master

Reply:
I am commenting here because you are still making changes to an unrelated file here, although only empty spaces this time.

Reply:
Thank you for this @luisramos0. I did clean up the master and pushed again, no _file partial this time.